multidict/_multilib/pair_list.h

```diff
@@ -199,30 +203,43 @@ pair_list_shrink(pair_list_t *list)
 
 
 static inline int
-_pair_list_init(pair_list_t *list, calc_identity_func calc_identity)
+_pair_list_init(pair_list_t *list, bool calc_ci_identity)
 {
+    PyObject *str_lower = PyUnicode_InternFromString("lower");
```

Comment:
Rather of caching string object "lower", it may be better to cache method object `str.lower`.

Reply:
`str.lower` can't work with `CallMethod`